### ISSUE TITLE
Error: No module named 'urlparse'

### ISSUE THREAD

**Issue by bartmika** [tkaemming/django-subdomains#55]
Hi, I am using **Django 1.9** with **Python 3.4** and when I try to run this library I get the following error:

```
File "/Users/xxx/Developer/rodolfomartinez/yyyy/py-zzzzz/env/lib/python3.4/site-packages/subdomains/utils.py", line 2, in <module>
    from urlparse import urlunparse
ImportError: No module named 'urlparse'
```

The solution to having this library support the missing library is to have this project import ``urllib`` and have **utils.py** have the following code:

```
try:
    from urllib.parse import urlparse
except ImportError:
    from urlparse import urlparse
```

**Comment by bartmika** [comment]
Upon investigation, it looks like when I run:

```
pip install django-subdomains 
```

When I check the version, it appears pip installs version 2.0.x which causes the error in my OP.

However, if I run:

```
pip install django-subdomains==2.1.0rc0
```

And then run the app and this error disappears. I don't know what happened, but I think this issue is considered closed unless you think there might be something to do with the configuration here.

**Comment by tkaemming** [comment]
This was due to fact that this was marked as release candidate and not a stable release. I fixed this with be6cc1c556a2007287ef4e647ea1784cf7690a44 of GH-46.
